packages/frontend-core/src/fetch/UserFetch.js

```diff
@@ -33,7 +33,7 @@ export default class UserFetch extends DataFetch {
     let finalQuery
     // convert old format to new one - we now allow use of the lucene format
     const { appId, paginated, ...rest } = query
-    if (!LuceneUtils.hasFilters(query) && rest.email) {
+    if (!LuceneUtils.hasFilters(query) && rest.email != null) {
```

Comment:
Is this intentionally a coercing check (`!=` vs `!==`)? What's the specific thing you're changing this for?

Reply:
yes, its the one singular time I'm ok with a `!=` rather than a specific check, as `!= null` checks for both of the empty states, null and undefined, whereas `!== null` will miss undefined.